### DISCUSSION TITLE
Sprint 0

### DISCUSSION THREAD

**Discussion by ekhenderson** [cs481-ekh/f22-kitty-cat-studios#22] [Instructor Feedback]
Team: I will use this area to give you feedback on your progress:

- I see the minutes for the Sprint 0 planning, although it looks a bit like a Discord log snapshot
- Please use the [ ] markdown for A/Is and check them off when completed
- I don't see a Sprint Goal for Sprint 0 (this either needs to be in the Meeting Minutes or on the sprint board), nor any user stories/tasks
- The PRs for project setup (assignment 1) should be added to the Sprint 0 backlog
- I see you have set the filters in your sprint tab views, good (I missed that in the instructions). But your Sprint 0 filter should be "sprint 0" not no:sprint
- I suppose you will wrap up Sprint 0 planning after meeting with Ben today

**Comment by Nathan-Turtleduck** [comment]
What does A/Is mean?

**Comment by ekhenderson** [comment]
A/Is = Action Items
